stdlib/2/os/__init__.pyi

```diff
@@ -130,7 +133,14 @@ def setuid(uid: int) -> None: ...  # Unix only
 def strerror(code: int) -> str: ...
 def umask(mask: int) -> int: ...
 def uname() -> Tuple[str, str, str, str, str]: ...  # Unix only
-def unsetenv(varname: str) -> None: ...
+
+@overload
+def getenv(key: Text) -> Optional[str]: ...
```

Comment:
Well let me know if you want me to change it. And also if you want me to stop merging modules into 2and3.  :)